core/src/OpenACC/Kokkos_OpenACC_Instance.cpp

```diff
@@ -58,31 +58,33 @@ namespace Kokkos {
 bool show_warnings() noexcept;
 }
 
-Kokkos::Experimental::Impl::OpenACCInternal*
+int Kokkos::Experimental::Impl::OpenACCInternal::m_accDev = -1;
```

Comment:
Will do.

Reply:
Actually I wonder if we should not set it to an arbitrary non-negative value that is unlikely to valid instead since a negative value tells the OpenACC runtime to revert to its (implementation-defined) default behavior

Reply:
A comment is added. 
(Strictly speaking, the OpenACC runtime may not complain an error even if the device number is negative (In OpenACC spec, how to handle negative device number is implementation-defined). However, the Kokkos runtime will issue an error.)

Reply:
I was going for
```C++
// Arbitrary value to denote that we don't know yet what device to use.
// The intent is that the value below is invalid.  The OpenACC runtime
// interprets negative values as "revert to (implementation-defined) default
// behavior".
int Kokkos::Experimental::Impl::OpenACCInternal::m_acc_device_num = 255;

Reply:
We actually do not know what the OpenACC runtime use to denote the host.

Reply:
In OpenACC, handling device number bigger that the number of available devices of a given type is also implementation-defined, and thus using an arbitrary big positive integer (255) will now solve the problem.